modules/removeBook.js

```diff
@@ -0,0 +1,4 @@
+export default function removeBook() {
```

Comment:
* [ ] Your function works perfectly, and that's great! As this project specifically requires the use of ES6 syntax, I would recommend utilizing arrow functions instead of regular functions. This practice aligns with the ES6 standard and enhances the readability and maintainability of the code. Please consider updating your code to use arrow functions wherever appropriate. Keep up the good work, and happy coding! 🚀🎉